### ISSUE TITLE
Any possibility that Godot will implement Basis Universal Texture Format?

### ISSUE THREAD

**Issue by earlroxas13** [godotengine/godot#29142]
**Issue description:**

It's a discussion. So a few days ago Google and Binomial open sourced Basis Texture format. From what I read in [their post](https://opensource.googleblog.com/2019/05/google-and-binomial-partner-to-open.html), i think it's a good texture format, because:
1. The size is half the .jpeg file.
2. VRAM usage is also several times more smaller than jpeg/png
3. The transcoder can transcode the basis file into format that GPU can read.
4. [Use Apache-2.0 Licence](https://github.com/binomialLLC/basis_universal), so, it should be compatible with Godot, right?

So, it can benefit the Godot because, it can lower the size from exported image files and also can reduce the GPU load. Also now that Godot have 2 renderers (GLES2 & GLES3), GLES2 must use ETC1 format, so it will be hassle if you make game that fallback from GLES3 and use ETC2 format. With this format you can just use basis format and let the transcoder decide to use ETC1 or ETC2 for you.

So what do you think? Should it be implemented? If not why not?
Or anyone here can implement it?

**Comment by Calinou** [comment]
This was discussed on IRC recently; search for `binomial` in the [logs](http://godot.eska.me/irc-logs/devel/2019-05-23.log).

**Comment by follower** [comment]
As an update to this request, see the "Basis Universal" section of "[Vulkan progress report #4](https://godotengine.org/article/vulkan-progress-report-4)":

> Basis Universal was added to Godot [4.0] as an optional import format.

**Comment by Calinou** [comment]
I guess we can close this once the `vulkan` branch is merged into `master` :slightly_smiling_face:

**Comment by akien-mga** [comment]
Well we can close it now, it's implemented already for the 4.0 milestone.
